transcom/mymove
# B-18173 Move history tracking for PPM weight docs

## [B-18173](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871867)

## Summary

This PR addresses the need to track a closeout counselors actions when reviewing weight tickets, whether they make adjustments or are approving or rejecting the weight. It also includes migrations and move_history_fetcher.sql changes for [B-18174](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871868) and [B-18175](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871869) as well; I chose to include that work in this PR so we wouldn't create 3 separate temp tables in 3 separate PRs and slow our query down.

### How to test

1. Login as a service counselor and view closeout PPMs
2. Review documents and make adjustments to the values while reviewing a weight ticket
3. Accept or reject your changes and save
4. Navigate to Move history and confirm your changes are reflected there
